kubernetes/README.md

```diff
@@ -464,8 +464,8 @@ Out-of-the-box, Kuma provides full integration with Prometheus and Grafana. If e
 In Kubernetes mode, we can use `kumactl install [..]` again to install the pre-configured Prometheus and Grafana components onto the Kubernetes cluster we have deployed:
 
 ```
-$ ./kumactl install metrics | kubectl apply -f -
-namespace/kuma-metrics created
+$ ./kumactl install observability --components grafana,prometheus | kubectl apply -f -
```

Comment:
Is it really worth limiting the components list and not install the whole stack?